### ISSUE TITLE
Enhancement: Berries and Balls

### ISSUE THREAD

**Issue by quindraco** [NECROBOTIO/NecroBot#4835]
Citation:  http://pokemongo.gamepress.gg/catch-mechanics

From what we can tell, berries multiply the capture rate by a constant *after* the rate has been modified by the ball, and the ball modification rate is quadratic in the unmodified rate (constant1*rate - constant2*rate^2, where the two constants are set by the ball).  This means the config options for setting when to use berries and balls are insufficient - we need, in general, a way to set all 6 options of when to use a ball, a berry, or both.  For example, here is the spread for ensuring your rate is always at least 50%, assuming the constants in the citation given are correct:

>=50.00%: Pokeball
>=38.20%: Greatball
>=33.33%: Pokeball + Berry
>=29.29%: Ultraball
>=24.17%: Greatball + Berry
>=18.35%: Ultraball + Berry
<18.35%:    Ultraball + Berry (rate will still be less than 50%)

To recap, I'm requesting the ability to convey the above logic to Necro, so rather than setting the min odds for berries and the min odds for each ball separately, I can set min odds for each of the six ball and berry *combinations*.
